components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -505,6 +505,26 @@ - (void)testBottomDrawerScrollingEnabled {
   XCTAssertEqual(self.fakeScrollView.scrollEnabled, NO);
 }
 
+- (void)testSetTrackingScrollView {
+  // Given
+  MDCBottomDrawerPresentationController *drawerPresentationController = nil;
+  if ([self.drawerViewController.presentationController
```

Comment:
Since this is a unit test, why can't we guarantee this precondition? Can you set it to be an `MDCBottomDrawerPresentationController` directly and then skip the conditional?

Reply:
Done